bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/IFindReplaceLogic.java

```diff
@@ -166,4 +166,15 @@ public interface IFindReplaceLogic {
 	 */
 	public IFindReplaceTarget getTarget();
 
+	/**
+	 * Returns <code>true</code> if searching should be restricted to entire words,
+	 * <code>false</code> if not. This is the case if the respective checkbox is
+	 * turned on, regex is off, and the checkbox is enabled, i.e. the current find
+	 * string is an entire word.
+	 *
+	 * @param findString the string that is currently being searched for.
+	 * @return <code>true</code> if the search is restricted to whole words
```

Comment:
```suggestion
	 * @return <code>true</code> if the search can be restricted to whole words
```

Reply:
Addressed in https://github.com/eclipse-platform/eclipse.platform.ui/commit/a0b224321663e63aa99f60da4a2527f993d68f7d